docs/api.md

```diff
@@ -51,6 +51,11 @@ The `SpatialNavigationNode` component receives the following props:
 | `onSelect`    | `function`                  | `undefined`  | Callback function to be called when the node is selected.                                                                                                                                                                                                     |
 | `orientation` | `'vertical' \| 'horizontal` | `'vertical'` | Determines the orientation of the node.                                                                                                                                                                                                                       |
 | `isFocusable` | `boolean`                   | `false`      | Determines if the node is focusable or not. If it's `true`, the `children` prop must be a function that returns a React element and accepts a parameter with a `isFocused` property. If it's `false` or not provided, `children` can be any valid React node. |
+
+| `alignInGrid` | `boolean`                   | `false`      | Determines whether child lists should behave like a grid.
+
+| `indexRange`  | `number[]`                  | `undefined`  | Determines the indexes when using long nodes in a grid. cf : https://github.com/bbc/lrud/blob/master/docs/usage.md#indexrange 
```

Comment:
You should mention that if a grid row has one `indexRange`, you should specify each element's `indexRange` (and refer to the example) 😊